src/components/grid/grid.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import customProperties from '../../../materials/custom-properties.json';
+import filterDataAttributes from '../../utils/filter-data-attributes';
+
+// TODO: check with design if those "breakpoints" are ok
+const minColumnSizes = {
+  1: '600px',
+  2: '450px',
+  3: '350px',
+  4: '275px',
```

Comment:
Should these be the same as our horizontal constraints?

Reply:
Hmm good point. I can check in the places where we use this already if the sizes make sense.